pmd-core/src/main/java/net/sourceforge/pmd/RuleSetFactory.java

```diff
@@ -848,15 +841,15 @@ private static String valueFrom(Node parentNode) {
     private static void parsePropertyNodeBR(Rule rule, Node propertyNode) {
 
         Element propertyElement = (Element) propertyNode;
-        String typeId = propertyElement.getAttribute(PropertyDescriptorFields.TYPE);
-        String strValue = propertyElement.getAttribute(PropertyDescriptorFields.VALUE);
+        String typeId = propertyElement.getAttribute(PropertyDescriptorField.TYPE.attributeName);
```

Comment:
I think so. I think it was related to PropertyDescriptorFactory (the old, bloated version in package `properties`), but was replaced with a better implementation... Although we can't really tell without any comments